packages/strapi-design-system/src/Combobox/utils.js

```diff
@@ -30,10 +30,13 @@ export const TreeActions = {
 // filter an array of options against an input string
 // returns an array of options that begin with the filter string, case-independent
 export function filterOptions(options = [], filter, exclude = []) {
-  return filter
+  const equalizedTerm = String(filter || '').toLowerCase();
```

Comment:
What do you think about using:

```suggestion
  const equalizedTerm = String(filter ?? '').toLowerCase();
```

instead?

```js
String(0 || '') // ''
String(0 ?? '') // '0'
```

Reply:
Awesome!
Thank you.